Test/Integration/Model/ConfigTest.php

```diff
@@ -47,10 +47,15 @@ public function testGetIdReturnsEmptyStringByDefault(): void
     }
 
     /**
-     * @magentoConfigFixture default/web/cookiebot/id 123-456-789
+     * @magentoConfigFixture current_store web/cookiebot/id 123-456-789
      */
     public function testGetIdReturnsId(): void
     {
         self::assertEquals('123-456-789', $this->config->getId());
     }
+
+    public function testEuCdnEnabledReturnsFalseByDefault(): void
```

Comment:
Similar to the other methods, we should also test if this method returns true if the config is true and returns false if the config is false.

Reply:
This test is for a completely different class. So yes, I think it makes sense to have those tests. Plus, they are incredibly easy to add.